plugins/org.eclipse.fordiac.ide.export.forte_ng/src/org/eclipse/fordiac/ide/export/forte_ng/base/BaseFBImplTemplate.xtend

```diff
@@ -153,4 +152,10 @@ abstract class BaseFBImplTemplate<T extends BaseFBType> extends ForteFBTemplate<
 			]
 		).toSet
 	}
+	
+	def private generateInternalFBInitializer() ///
+	'''«FOR fb : type.internalFbs BEFORE ",\n" SEPARATOR ",\n"»«FB_EXPORT_PREFIX»«fb.name»(«fb.name.FORTEStringId», *this)«ENDFOR»'''
+	
+	def private generateAddInternalFB() ///
+	'''«FOR fb : type.internalFbs SEPARATOR "\n"»addFB(&«FB_EXPORT_PREFIX»«fb.name»);«ENDFOR»'''
```

Comment:
Please use `fb.generateName` instead of `«FB_EXPORT_PREFIX»«fb.name»`.

Reply:
Done